gamemode/modules/base/sh_util.lua

```diff
@@ -34,17 +34,24 @@ function DarkRP.formatMoney(n)
     if n >= 1e14 then return attachCurrency(tostring(n)) end
     if n <= -1e14 then return "-" .. attachCurrency(tostring(math.abs(n))) end
 
+    local config = GAMEMODE.Config
+
     local negative = n < 0
 
     n = tostring(math.abs(n))
-    local dp = string.find(n, "%.") or #n + 1
+
+    if config.currencyDecimalSeparator ~= "." then
+        n = string.Replace(n, ".", config.currencyDecimalSeparator)
+    end
+
+    local dp = string.find(n, config.currencyDecimalSeparator, 1, true) or #n + 1
 
     for i = dp - 4, 1, -3 do
-        n = n:sub(1, i) .. "," .. n:sub(i + 1)
+        n = n:sub(1, i) .. config.currencyThousandSeparator .. n:sub(i + 1)
     end
 
     -- Make sure the amount is padded with zeroes
-    if n[#n - 1] == "." then
+    if n[#n - 1] == config.currencyDecimalSeparator then
```

Comment:
In light of not supporting decimals, I think this code is dead in DarkRP. It should be possible to remove it.